src/site/content/jakarta-ee.adoc

```diff
@@ -0,0 +1,387 @@
+= Apache Shiro Jakarta EE Integration
+:jbake-date: 2023-02-06 00:00:00
+:jbake-type: page
+:jbake-status: published
+:jbake-tags: documentation, jakarta-ee, integrations, web
+:idprefix:
+:icons: font
+
+Apache Shiro Jakarta EE module makes it transparent to use Shiro features in Jakarta EE applications
+with minimal configuration. It makes annotations such as `@RequiresRoles` available in Jakarta EE (CDI, EJB, etc.) code.
+
+NOTE: Jakarta EE integration is available in Shiro 2.0 or later.
```

Comment:
Does it make sense to note which EE versions are compatible to the module? (as the whole thing was rebranded and we have the javax vs jakarta namespace thing)

Reply:
➕ 1️⃣

Reply:
done